cmd/analyzer/analyzer.go

```diff
@@ -84,11 +107,32 @@ func parseFlags() cliFlags {
 	if cli.shardPrefixesFlag != "" {
 		cli.shardPrefixes = strings.Split(cli.shardPrefixesFlag, ",")
 	}
+
+	if cli.json {
+		data := &MistConfig{
+			Name:         "LivepeerAnalyzer",
+			Version:      version,
+			Description:  "",
```

Comment:
Couldn't think of a one-line description to put here.

Reply:
Hmmm, could be something in the lines of 
`"Stream health data analyzer service for aggregated reporting of running streams status"`

Reply:
done. thanks :)